### PR TITLE
Improve the behavior of untargeted MIRV shots

### LINKED ISSUE #293: Improve default behavior of MIRV weapons (namely scylla)
Ideas:
* Pentagon shape
* Default all at one position
* Circle spread tightened

### PR DESCRIPTION
**Pull request by HTMLPhoton**
this resolves #293

### CODE REVIEW

**Review by Sharkgamedev — APPROVED** [review]

**Comment by Sharkgamedev** [comment]
> this resolves #293

Everything looks good, from the Discord it seems that Shaman is taking a break or something else. I figure I could contribute my review to help keep things going while he's gone? I'm not sure if you also have push access, but I'm able to merge this if not.

File: LuaRules/Gadgets/weapon_cluster_ammunition_spawner.lua
Comment (Sharkgamedev) [inline LuaRules/Gadgets/weapon_cluster_ammunition_spawner.lua:246]: This is old code moved up by the looks? If you know what the magic number is maybe add a comment describing it or otherwise?

**Comment by Arch-Shaman** [comment]
> Everything looks good, from the Discord it seems that Shaman is taking a break or something else. I figure I could contribute my review to help keep things going while he's gone? I'm not sure if you also have push access, but I'm able to merge this if not.

I am indeed on break at the moment. Though Ill merge this. Currently finishing up verbs in Finnish and training pronunciation. Ive gotten to the point of making consistent sentences asynchronously.

**Pull request merged**
